charts/sonic/templates/statefullSet.yaml

```
@@ -0,0 +1,86 @@
+
+apiVersion: apps/v1beta2
+kind: StatefulSet
+metadata:
+  name: {{ include "sonic-helm-chart.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "sonic-helm-chart.name" . }}
+    helm.sh/chart: {{ include "sonic-helm-chart.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "sonic-helm-chart.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "sonic-helm-chart.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
+    spec:
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          command: ["sonic" ,"-c", "/etc/config/sonic.cfg"]
+          volumeMounts:
+          {{- if .Values.persistence.enabled }}
+            - name: data
+              mountPath: /var/lib/sonic/store/
```

Shouldn't it be?

```suggestion
              mountPath: /usr/src/sonic/data
```

because in config is `path = "./data/store/kv/"` and workdir is [/usr/src/sonic](https://github.com/valeriansaliou/sonic/blob/f5302f5c424256648ba0be32b3c5909d846821fe/Dockerfile#L20)